ansible_collections/arista/avd/roles/eos_designs/templates/backbone_interfaces/router-ospf.j2

```diff
@@ -0,0 +1,7 @@
+{% if backbone_interfaces_data.ospf_interfaces | length > 0 %}
```

Comment:
Fixed. Unless I'm mistaken backbone interfaces settings do not need to include any info about the ospf process ID since it's not really configurable on the interface level, you just go "ip ospf area X" and it'll get included in whichever ospf process is running for that VRF.